exporters/otlp/otlplog/otlplogfile/config.go

```diff
@@ -0,0 +1,54 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package otlplogfile // import "go.opentelemetry.io/otel/exporters/otlp/otlplog/otlplogfile"
+
+import "time"
+
+type fnOpt func(config) config
+
+func (f fnOpt) applyOption(c config) config { return f(c) }
+
+// Option sets the configuration value for an Exporter.
+type Option interface {
+	applyOption(config) config
+}
+
+// config contains options for the OTLP Log file exporter.
+type config struct {
+	// Path to a file on disk where records must be appended.
+	// This file is preferably a json line file as stated in the specification.
+	// See: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/file-exporter.md#json-lines-file
+	// See: https://jsonlines.org
+	path string
+	// Duration represents the interval when the buffer should be flushed.
+	flushInterval time.Duration
+}
+
+func newConfig(options []Option) config {
+	c := config{
+		path:          "/var/log/opentelemetry/logs.jsonl",
```

Comment:
There should be options to write to standard output and standard error.

The standard output should be the default output.

Related PR: https://github.com/open-telemetry/opentelemetry-specification/pull/4183

Reply:
Sure, I've added this to my latest commit. I added a `WithWriter()` option so that users can easily use any concrete implementation of `io.WriteCloser`. By default, `os.Stdout` is used as requested. 

For convenience, I also added a `WithFile()` option, that tries to create or open the file at the given path. Maybe this is out of scope and `WithWriter()` is sufficient?

Reply:
`WithWriter` is sufficient. I added comments in my review. Resolving this one.